src/test/java/test/DynamicGraphTest.java

```diff
@@ -126,4 +139,71 @@ public void test2() {
     assertFreeNodesEquals(dg);
   }
 
+  @Test
+  public void testOrderingOfEdgesWithSameWeight() {
+    XmlSuite xmlSuite = new XmlSuite();
+    xmlSuite.setName("1360_Suite");
+    Class<?>[] classes = new Class[] {
+        TestClassContainerForGitHubIssue1360.TestNG1.class,
+        TestClassContainerForGitHubIssue1360.TestNG2.class,
+        TestClassContainerForGitHubIssue1360.TestNG3.class
+    };
+    XmlTest xmlTest = createXmlTest(xmlSuite, "1360_Tests", classes);
+    IAnnotationFinder annotationFinder = new JDK15AnnotationFinder(new DefaultAnnotationTransformer());
+    List<ITestNGMethod> methods = Lists.newArrayList();
+    for (Class<?> clazz : classes) {
+      methods.addAll(
+          Arrays.asList(
+              AnnotationHelper.findMethodsWithAnnotation(clazz, ITestAnnotation.class, annotationFinder, xmlTest)
+          )
+      );
+    }
```

Comment:
I really love the hack you made! :D 
Could you extract it in a method? Something like `List<ITestNGMethod> extractTestNGMethods(Class<?>... classes)`